eZ/Bundle/PlatformInstallerBundle/src/Resources/config/database.yml

```diff
@@ -0,0 +1,18 @@
+services:
+    ezplatform.installer.database.connection.factory:
+        class: EzSystems\PlatformInstallerBundle\Database\ConnectionFactory
+        arguments: ["@ezpublish.persistence.connection"]
+
+    ezplatform.installer.database.connection:
+        class: Doctrine\DBAL\Connection
+        factory: ["@ezplatform.installer.database.connection.factory", createConnection]
+
+    ezplatform.installer.database.platform.mysql:
```

Comment:
Okay. What bothers me is that even if "Platform" has a meaning in the Doctrine domain, it also has one in our domain, and it is confusing to me.

Could we find an alternative ?